src/human-paddle-control/Dockerfile.zscaler

```diff
@@ -55,10 +58,8 @@ RUN curl -sSf https://librealsense.intel.com/Debian/librealsense.pgp | sudo tee
 RUN sudo apt-get install apt-transport-https -y
 RUN echo "deb [signed-by=/etc/apt/keyrings/librealsense.pgp] https://librealsense.intel.com/Debian/apt-repo `lsb_release -cs` main" | sudo tee /etc/apt/sources.list.d/librealsense.list
 RUN sudo apt-get update
```

Comment:
do you not need the '--fix-missing' here also?

Reply:
Likely a good idea. The other --fix-missing was added due to a weird likely ephemeral build issue